File: Ohtap E Commerce Domain Models/Development/PHP/Objects/PackageAttributes/PackageAttribute.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\PackageAttributes;

class PackageAttribute  implements ILabel
{

    public $ID;
    public $Details;//AttributeDetailDataList
    public $Attribute;//Attribute
    public $State;
    public $Value;
    public $OrderNo;
    public $Description;
}

File: Ohtap E Commerce Domain Models/Development/NetCore/Models Objects/Models Objects/PackageAttributes/PackageAttribute.cs
using Ohtap.ECommerce.Domain.Models.Objects.Products;
using Ohtap.ECommerce.Domain.Models.Objects.PackageDetails;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.PackageAttributeDetails;

namespace Ohtap.ECommerce.Domain.Models.Objects.PackageAttributes
{
    public class PackageAttribute : PackageAttributeBasic, IPackageAttribute
    {
        private PackageAttributeDetailList _Details;
        public virtual PackageAttributeDetailList Details { get => base.GetProperty(this._Details); set => base.SetProperty(ref this._Details, value); }
        IPackageAttributeDetailDataList<IPackageAttributeDetailData> IPackageAttribute.Details { get { return this.Details as IPackageAttributeDetailDataList<IPackageAttributeDetailData>; } set { this.Details = (PackageAttributeDetailList)value; } }
    }
}
